geode-wan/src/main/java/org/apache/geode/internal/cache/wan/GatewayReceiverImpl.java

```diff
@@ -164,13 +164,12 @@ public void start() throws IOException {
       } catch (BindException be) {
         throw new GatewayReceiverException(
             LocalizedStrings.SocketCreator_FAILED_TO_CREATE_SERVER_SOCKET_ON_0_1
-                .toLocalizedString(new Object[] {bindAdd, this.port}));
+                .toLocalizedString(bindAdd, this.port), be);
```

Comment:
I think it would be better to not include the entire exception as a cause but doing so is better than what you had before. I think it would be better to call getMessage on be and just include that as one of the arg to the localized string.